sync_known_issues.py

```diff
@@ -196,6 +198,32 @@ def parse_files(config_files):
                 sys.exit(1)
     return config_data
 
+def issues_equal(a, b):
+    """
+        Compare two known issue dictionaries,
+        return True if equal, else False
+    """
+
+    # Copy the dicts, so they may be modified
+    x = a.copy()
+    y = b.copy()
+
+    # Copy and remove 'id' for purpose of comparison
+    # Remove 'id' field
+    if 'id' in x: del x['id']
+    if 'id' in y: del y['id']
+
+    # Sort lists for purpose of comparison
+    x['environments'].sort()
+    y['environments'].sort()
```

Comment:
I'm a bit confused on how it works. Locally environments are in form of 'some_env_name'. Remotely SQUAD presents environments as 'https://squad.tld/api/environmnets/ID'. How do these compare without resolving the url to the object?